docs/tutorial/advanced.rst

```diff
@@ -1,31 +1,183 @@
+When gold standards are provided and evaluation is enabled (include: true), SPRAS will automatically compare the reconstructed subnetworks for a specific dataset against the corresponding gold standards.
+
+.. code-block:: yaml
+
+    gold_standards:
+        - 
+        label: gs1
+        node_files: ["gs_nodes0.txt", "gs_nodes1.txt"]
+        data_dir: "input"
+        dataset_labels: ["data0"]
+        - 
+        label: gs2
+        edge_files: ["gs_edges0.txt"]
+        data_dir: "input"
+        dataset_labels: ["data0", "data1"]
+
+    analysis:
+        evaluation:
+        include: true
+
+A gold standard dataset must include the following types of keys and files:
+
+- label: a name that uniquely identifies a gold standard dataset throughout the SPRAS workflow and outputs.
+- node_file or edge_file: A list of node or edge files. Only one of these can be defined per gold standard dataset.
+- data_dir: The file path of the directory where the input gold standard dataset files are located.
+- dataset_labels: a list of dataset labels indicating which datasets this gold standard dataset should be evaluated against.
+
+When evaluation is enabled, SPRAS will automatically run its built-in evaluation analysis on each defined dataset-gold standard pair.
+This evaluation computes metrics such as precision, recall, and precision-recall curves, depending on the parameter selection method used.
+
+For each pathway, evaluation can be run independently of any parameter selection method (the ground truth-based evaluation without parameter selection idea) to directly inspect precision and recall for each reconstructed network from a given dataset.
+
+.. image:: ../_static/images/pr-per-pathway-nodes.png
+   :alt: description of the image
+   :width: 400
+   :align: center
+
+.. raw:: html
+
+   <div style="margin:20px 0;"></div>
+
+Ensemble-based parameter selection generates precision-recall curves by thresholding on the frequency of edges across an ensemble of reconstructed networks for an algorithm for given dataset.
+
+.. image:: ../_static/images/pr-curve-ensemble-nodes-per-algorithm-nodes.png
+   :alt: description of the image
+   :width: 400
+   :align: center
+
+.. raw:: html
+
+   <div style="margin:20px 0;"></div>
+
+PCA-based parameter selection computes a precision and recall for a single reconstructed network selected using PCA from all reconstructed networks for an algorithm for given dataset.
+
+.. image:: ../_static/images/pr-pca-chosen-pathway-per-algorithm-nodes.png
+   :alt: description of the image
+   :width: 400
+   :align: center
+
+.. raw:: html
+
+   <div style="margin:20px 0;"></div>
+
+.. note:: 
+    Evaluation will only execute if ml include is also set to true, because the PCA parameter selection step depends on the PCA ML analysis.
+
+.. note:: 
+    To see evaluation in action, run SPRAS using the config.yaml or egfr.yaml configuration files.
 
 CHTC integration
```

Comment:
I don't think we should call this CHTC. Maybe HTCondor or High-throughput computing. CHTC is specific to one institution.